ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -864,6 +864,13 @@ def cpu(self):
         self.ivy_array = ivy.to_device(self.ivy_array, ivy.as_ivy_dev("cpu"))
         return self
 
+    @with_unsupported_dtypes(
+        {"2.5.1 and below": ("int16", "complex64", "complex128")},
+        "paddle",
+    )
+    def split(self, num_or_sections, axis=0, name=None):
+        return paddle_frontend.split(self._ivy_array, num_or_sections, axis, name)
+
```

Comment:
Hi @Mac16661, the method looks good (you don't really need to pass `name` though, it's not used). 
Now you just need to write the test for it.

Reply:
And Anna can u tell me about this failed tests and how to pass it. One more thing I am not sure how to edit the PR description to link to the issue I have created.

Reply:
should I write in "Close #26495" in the comment section 

[ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues)->
reference the issue in the pull request by adding the following content to the description of the PR:
Close #Issue_number

Reply:
You just have to edit the top PR comment adding "Close #issue_number" like I did just now. However issue #26495 refers to `index_sample` while you are adding `split`.

Reply:
You will need to add your test and work on making it pass. Any failures seen right now in the CI checks are irrelevant to your PR and you don't need to worry about them.

Reply:
Thanks @AnnaTz